### PR TITLE
feat(locale): update fa's word module

### PR DESCRIPTION
**Pull request by ParsaArvanehPA**
Updated the fa's word module to be exactly like en's word module.

### CODE REVIEW

File: src/locales/fa/word/adjective.ts
Comment (ST-DDT) [inline src/locales/fa/word/adjective.ts:1]: Please limit this file to 1000 entries.

File: src/locales/fa/word/verb.ts
Comment (ST-DDT) [inline src/locales/fa/word/verb.ts:1]: Please limit this file to 1000 entries.

**Comment by matthewmayer** [comment]
@ParsaArvanehPA please also check for duplicate strings in each definition file for example `اوه` in the interjections. The tests will fail if there are any duplicate strings in a file. You can run `pnpm run preflight` locally to check this.

File: src/locales/fa/word/verb.ts
Comment (ParsaArvanehPA) [reply src/locales/fa/word/verb.ts:1]: I will get back to you with the requested changes as soon as I can.

**Comment by ST-DDT** [comment]
@ParsaArvanehPA Could you please address the comments?
If you don't have time please close this PR. You can still reopen it later, when you have time.
We will soon begin development for v9 and would like to resolve all open PRs for that.

**Comment by ParsaArvanehPA** [comment]
> @ParsaArvanehPA Could you please address the comments? If you don't have time please close this PR. You can still reopen it later, when you have time. We will soon begin development for v9 and would like to resolve all open PRs for that.

Oh I'm really sorry I completely forgot about this PR; for now, like you suggested, I will close it.

**Pull request closed without merging**